nessie/src/test/java/org/apache/iceberg/nessie/TestBranchVisibility.java

```diff
@@ -122,10 +119,11 @@ private void updateSchema(NessieCatalog catalog, TableIdentifier identifier) {
     catalog.loadTable(identifier).updateSchema().addColumn("id" + schemaCounter++, Types.LongType.get()).commit();
   }
 
-  private void testCatalogEquality(NessieCatalog catalog,
-                                   NessieCatalog compareCatalog,
-                                   boolean table1Equal,
-                                   boolean table2Equal) {
+  private void testCatalogEquality(
+      NessieCatalog catalog,
+      NessieCatalog compareCatalog,
+      boolean table1Equal,
+      boolean table2Equal) {
```

Comment:
I think the project prefers keeping multiple arguments on a line and wrapping at the same depth as the starting argument. So there are multiple options:

```java
  private void testCatalogEquality(NessieCatalog catalog, NessieCatalog compareCatalog,
                                   boolean table1Equal, boolean table2Equal) {
```

or,

```java
  private void testCatalogEquality(
      NessieCatalog catalog, NessieCatalog compareCatalog, boolean table1Equal, boolean table2Equal) {
```

I think one thing that we don't usually want is one argument per line since you end up having really long argument lists. That's a problem in both cases, though.

Is this something we can fix in the style config that you've applied?

Reply:
I don't prefer having each argument on a separate line either, but that is the result of formatting the code with `intellij-java-palantir-style.xml`. As mentioned on #2732 it would be ideal to enforce such style rules so that unformatted code doesn't make it into the project

Reply:
seems like the formatter defaults to putting every single argument into a new line when **all** of them don't fit into the same line. However, if one does split up the args manually up and format the code, then the below format is valid as well. It's just that this preferrable format isn't automatically the first choice of the formatter and I'm not sure we can do anything about it atm.

```
  private void testCatalogEquality(
      NessieCatalog catalog, NessieCatalog compareCatalog, boolean table1Equal, boolean table2Equal) {
```